src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/list/main.js

```diff
@@ -9,20 +9,13 @@
 
 define([
     'mvc/relationalstore',
-    'app-config'
-], function(RelationalStore, AppConfig) {
+    'app-config',
+    'widget-groups'
+], function(RelationalStore, AppConfig, WidgetGroups) {
 
     'use strict';
 
     var bindCustomEvents = function() {
-            // navigate to edit account
-            this.sandbox.on('husky.datagrid.item.click', function(id) {
-                this.sandbox.emit(
-                    'sulu.sidebar.set-widget',
-                    '/admin/widget-groups/account-info?account=' + id
-                );
-            }, this);
```

Comment:
If you remove these lines, you probably also don't need the WidgetGroups, right?